src/main/java/net/gegy1000/zoomlayer/Shuffleable.java

```diff
@@ -0,0 +1,11 @@
+package net.gegy1000.zoomlayer;
+
+/**
+ * Provides an access to the layer's mixSeed command without the overhead of floorMod.
+ *
+ * @author SuperCoder79
+ */
+public interface Shuffleable {
+
+	void shuffle();
```

Comment:
Hmm, I feel like shuffle might have confusing connotations? As in it’s not so clear that it has the same effect as nextInt

Reply:
yeah, that's a good point. What should I call it instead?

Reply:
`skipInt` maybe?

Reply:
Although, maybe that's not clear enough